make/photon/registry/Dockerfile

```diff
@@ -2,14 +2,9 @@ ARG harbor_base_image_version
 ARG harbor_base_namespace
 FROM ${harbor_base_namespace}/harbor-registry-base:${harbor_base_image_version}
 
-COPY ./make/photon/common/install_cert.sh /home/harbor
-COPY ./make/photon/registry/entrypoint.sh /home/harbor
-COPY ./make/photon/registry/binary/registry /usr/bin/registry_DO_NOT_USE_GC
-
-RUN chown -R harbor:harbor /etc/pki/tls/certs \
-    && chown harbor:harbor /home/harbor/entrypoint.sh && chmod u+x /home/harbor/entrypoint.sh \
-    && chown harbor:harbor /home/harbor/install_cert.sh && chmod u+x /home/harbor/install_cert.sh \
-    && chown harbor:harbor /usr/bin/registry_DO_NOT_USE_GC && chmod u+x /usr/bin/registry_DO_NOT_USE_GC
+COPY --link --chown=10000:10000 --chmod=755 ./make/photon/common/install_cert.sh /home/harbor/
+COPY --link --chown=10000:10000 --chmod=744 ./make/photon/registry/entrypoint.sh /home/harbor/
```

Comment:
I don't know why this is the case, but in the current image, the permission of `entrypoint.sh` is `744`.
The permission of `entrypoint.sh` is `644` on git, and execute `chmod u+x`(= +100) on it, so the result is `744`.
I don't think changing the permissions of `entrypoint.sh` to `755` will cause any problems, but I'm not sure.

Note that the permission of `install_cert.sh` is `755` on git, so it will be `755` whether or not `chmod u+x` is executed.

Reply:
In case any regression problem, I prefer to keep the original writing. But I agree that we optimize the docker file to reduce size.

Reply:
If we are willing to use a bind mount, we can do the following.
I have confirmed on local environment that image size can be reduced this way.
```
RUN --mount=type=bind,source=make/photon,target=/root \
    chown -R harbor:harbor /etc/pki/tls/certs \
    && cp /root/common/install_cert.sh /home/harbor \
    && cp /root/registry/entrypoint.sh /home/harbor \
    && cp /root/registry/binary/registry /usr/bin/registry_DO_NOT_USE_GC \
    && chown harbor:harbor /home/harbor/entrypoint.sh && chmod u+x /home/harbor/entrypoint.sh \
    && chown harbor:harbor /home/harbor/install_cert.sh && chmod u+x /home/harbor/install_cert.sh \
    && chown harbor:harbor /usr/bin/registry_DO_NOT_USE_GC && chmod u+x /usr/bin/registry_DO_NOT_USE_GC
```
Would this method be better?

Reply:
I think this way looks more clean and similar with previous.

Reply:
Pushed.